imsv-docs-astro/src/content/docs/guides/kyc/immersve-conducted-kyc.mdoc

```diff
@@ -87,7 +87,44 @@ curl -X POST "https://test.immersve.com/api/spending-prerequisites" \
   ]
 }
 ```
+### (Optional) Supply Contact Details
```

Comment:
This looks good! Only feedback is, I wonder if we should have (Optional) at the end of the title, like Supply Contact Details (Optional), or maybe even mention in the first paragraph that it's optional?

Reply:
at the end sure. i figure most dev's skim things, so making sure it's clear that is an optional step in the heading seemed to make the most sense.